src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlServer/00000000000000_CreateIdentitySchema.Designer.cs

```diff
@@ -17,14 +17,13 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.2.0-preview1")
+                .HasAnnotation("ProductVersion", "3.0.0-preview8")
                 .HasAnnotation("Relational:MaxIdentifierLength", 128)
                 .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
             modelBuilder.Entity("Microsoft.AspNetCore.Identity.IdentityRole", b =>
                 {
-                    b.Property<string>("Id")
-                        .ValueGeneratedOnAdd();
+                    b.Property<string>("Id");
```

Comment:
@bricelam Is it expected that we would lose `ValueGeneratedOnAdd` here?

Reply:
Not expected

Reply:
I figured this one out--we no longer make string keys value generated by default. See https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/breaking-changes#string-and-byte-array-keys-are-not-client-generated-by-default

@divega @HaoK @blowdart I think we should keep this change because:
* It doesn't change the schema
* The reason we made the change applies here as much as anywhere else
* It's unlikely to impact any real apps, and the fix is easy if it does

Reply:
So do we add it back? An ID string that doesn't generate IDs by default is unexpected

Reply:
Actually, from an EF perspective, a string key that generates IDs is very unexpected. EF has no knowledge of what king of string you would want. Now it just happens that we generate a Guid, and since Identity has this super-weird pattern of string keys that are actually GUIDs, then it might even work for Identity. But if that's the case, then it should be added back into Identity.

Reply:
In other words, Identity should not be inheriting an EF behavior here. If Identity needs generated keys, then that should be specified by Identity.

Reply:
So from the 'default' POCO perspective, we always generate the ID in the constructor via a new guid: https://github.com/aspnet/AspNetCore/blob/master/src/Identity/Extensions.Stores/src/IdentityRole.cs#L21

So does that mean we can just remove the ValueGeneratedOnAdd?  I think its reasonable to expect callers to specify the ID if they use their own POCOs

Reply:
Agreed. I think the EF model now fits better with the way it is used by Identity.

Reply:
Agreed.